ftasks/ftask2/ftask2.py

```diff
@@ -0,0 +1,97 @@
+import os
+import sys
+import json
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument("data_path", help="Phonebook file path")
+args = parser.parse_args()
+
+current_dir = os.path.dirname(os.path.realpath(__file__))
+
+PHONEBOOK = {}
+
+datafile_path = os.path.join(current_dir, (args.data_path + ".json"))
+
+
+def user_exit():
+    sys.exit()
```

Comment:
I think you shouldn't have created this function, just run `sys.exit()` in its place